src/Trixi.jl

```diff
@@ -213,6 +214,12 @@ function __init__()
   @require Plots="91a5bcdd-55d7-5caf-9e0b-520d859cae80" begin
     using .Plots: plot, plot!, savefig
   end
+
+  @require Makie="ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" begin
+    include("visualization/makie_visualization.jl")
+    using .Makie: Makie, Attributes, mesh, mesh!, plot!, @recipe, lines!, @lift
+    export trixi_plot
```

Comment:
They do allow user/type recipes. The type recipes hook into existing plot functions, and Makie.jl doesn't use `plot` as an actual plot signature. My understanding was that the user recipes let you define new plot commands (e.g., `myplot(customType)`, but I don't think the recipe enables `Makie.plot(customType)`. 

I'll check on this - it would be nice to have a unified `plot(sol)` interface.

Reply:
Looks like we can overload `Makie.plot` after all. Let me try to add a user recipe for `TrixiODESolution` in this PR.

Reply:
Sounds interesting :+1: Using the setup you currently have is also okay for me - I forgot that Makie uses a different interface than other plotting packages